en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -334,6 +334,103 @@ When there are null values in custom expressions and are:
 * Fields without attributes, the null value is ignored. If the value in a required field, an error message occurs. If the value is in an optional field, the value is ignored and not sent to the connector. +
 * Fields with attributes, the value is ignored and not sent to the connector.
 
+== Define a Variable
+
+You can define variables in your Composer flow. Defining a variable allows you to complete certain use cases like aggregating calculations and storing values. Variables are defined locally within a flow and can be used anywhere within that flow. 
+
+[[access-the-variable-step]]
+=== Access the Variable Step 
+
+To access the *Variable* step: 
+
+. <<create-a-new-flow,Create a new flow>> or open an existing flow. +
+The *<Flow Name>* page appears.
+. Click *Edit*. 
+. Locate the position in your flow where you want access the *Variable* step and click the plus button. 
+. In the *Next Step* window, click *Variable*. +
+The *Variable* step appears. 
+
+=== Create a Variable Using the Visual Editor 
+
+To create one or more new variables using the *Visual Editor* window: 
+
+. <<access-the-variable-step,Access the *Variable* step.>> 
+. In the *Action* dropdown, click *Create Variable*.
+. In the *Variable* section, click *Add Variable*. 
+. In the *Visual Editor*, in the *Label* field, enter a unique name for the variable. Note that the name entered is only unique to this step. Values in the *Label* field can be duplicated across different Variable steps. 
+. In the *Data Type* drop-down, select the data type you want to use for the variable. The following xref:ms_composer_custom_expression_editor.adoc#data-types[data types] are supported: `String`, `Number`, `Boolean`, `Date`, `DateTime`.
+. Optionally, to add additional variables in the *Visual Editor*, click *Add Variable*. You can create multiple variables in each Variable step. 
```

Comment:
"add additional" - can you remove "additional"?

Reply:
I guess I can because I have "multiple" in the next sentence.